radixdlt-core/radixdlt/src/main/java/com/radixdlt/consensus/sync/BFTSync.java

```diff
@@ -369,7 +371,8 @@ private void processGetVerticesLocalTimeout(VertexRequestTimeout timeout) {
 		}
 
 		if (syncRequestState.authors.isEmpty()) {
-			throw new IllegalStateException("Request contains no authors except ourselves");
+			log.warn("Request contains no authors except ourselves");
```

Comment:
This is a valid `IllegalStateException` since `VertexRequestTimeout` is a local message.